nix/shell.nix

```diff
@@ -22,7 +22,7 @@ mkShell {
   nativeBuildInputs = with ocamlPackages; [
     ocamlformat
     utop
-    ocaml-lsp
+    # ocaml-lsp
```

Comment:
intended?

Reply:
This wasn't intended, thanks for noticing!